js/application-settings-controller.js

```diff
@@ -279,15 +278,6 @@ function drawCylinderProject(){
   workspace.add(aCylinder)
   }
 }
-  
```

Comment:
@kyleines  Why was this removed?

Reply:
I had to remove the function calls here and put one in viewer.js to make the Project checkbox work correctly.